Add Faucet Gif (#35)
datahaven-xyz/datahaven-docs: store-and-retrieve-data/starter-kit.md

|      Testnet      |                           Where To Get Tokens From                           | Amount Dispensed                                                                 |
|:-----------------:|:----------------------------------------------------------------------------:|----------------------------------------------------------------------------------|
| DataHaven Testnet | [Official Testnet Faucet](https://apps.datahaven.xyz/testnet/faucet){target=\_blank} | The faucet dispenses {{ networks.testnet.website_faucet_amount }} every 12 hours |

> To get Testnet MOCK tokens, connect your wallet to the [Official Testnet Faucet](https://apps.datahaven.xyz/testnet/faucet){target=\_blank} and sign a message to login to the dApp. Then, navigate to the faucet page and press **Request Tokens** as shown below:

Reviewer: ```suggestion
To get testnet MOCK tokens:

1. Connect your wallet to the [Official Testnet Faucet](https://apps.datahaven.xyz/testnet/faucet){target=\_blank}.
2. Sign a message to login to the dApp.
3. From the faucet page, press **Request Tokens**.
```

|      Testnet      |                           Where To Get Tokens From                           | Amount Dispensed                                                                 |
|:-----------------:|:----------------------------------------------------------------------------:|----------------------------------------------------------------------------------|
| DataHaven Testnet | [Official Testnet Faucet](https://apps.datahaven.xyz/testnet/faucet){target=\_blank} | The faucet dispenses {{ networks.testnet.website_faucet_amount }} every 12 hours |

To get testnet MOCK tokens:

1. Connect your wallet to the [Official Testnet Faucet](https://apps.datahaven.xyz/testnet/faucet){target=\_blank}.
2. Sign a message to log in to the dApp.
3. From the **Faucet** page, press **Request Tokens**.

> ![Requesting tokens from the testnet faucet](images/store-and-retrieve-data/starter-kit/starter-kit-01.gif)

Reviewer: ```suggestion
![Requesting tokens from the testnet faucet](/images/store-and-retrieve-data/starter-kit/starter-kit-01.gif)
```

haha sorry, ideally all my comments would have been in one review, but it's monday 🤷 😬

|:-----------------:|:----------------------------------------------------------------------------:|----------------------------------------------------------------------------------|

> | DataHaven Testnet | [Official Testnet Faucet](https://apps.datahaven.xyz/faucet){target=\_blank} | The faucet dispenses {{ networks.testnet.website_faucet_amount }} every 12 hours |

Reviewer 2: ```suggestion
| DataHaven Testnet | [Official Testnet Faucet](https://apps.datahaven.xyz/testnet/faucet){target=\_blank} | The faucet dispenses {{ networks.testnet.website_faucet_amount }} every 12 hours |
```

To get TestNet HAVE tokens, connect your wallet to the [Official Testnet Faucet](https://apps.datahaven.xyz/faucet){target=\_blank} and sign a message to login to the dApp. Then, navigate to the faucet page and press **Request Tokens** as shown below:

![Requesting tokens from the testnet faucet](/images/faucet.gif)

> TestNet HAVE tokens have no market value. Please do not spam the faucet with unnecessary requests.

Reviewer 2: ```suggestion
Testnet MOCK tokens have no market value. Please do not spam the faucet with unnecessary requests.
```
